src/RetryProxy.php

```diff
@@ -89,4 +94,9 @@ public function call(callable $action, array $arguments = []): ?mixed
 
         throw new RetryException('Action call is failed.');
     }
+
+    public function getTryCount(): int
```

Comment:
perhaps add to RetryProxyInterface ? otherwise a new dependency will be introduced in the code using this?

Reply:
:-) I knew that was coming.  diky, opraveno.